src/track/serato/tags.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "track/serato/markers.h"
+#include "track/serato/markers2.h"
+
+namespace mixxx {
+
+// DTO for storing information from the SeratoMarkers_/2 tags used by the
+// Serato DJ Pro software.
+//
+class SeratoTags final {
+  public:
+    static constexpr RgbColor kDefaultTrackColor = RgbColor(0xFF9999);
+    static constexpr RgbColor kDefaultCueColor = RgbColor(0xCC0000);
+
+    SeratoTags() = default;
+
+    static RgbColor colorToStoredTrackColor(RgbColor::optional_t color);
```

Comment:
I would prefer the _encode_/_decode_ terminology, because this is what actually happens here.

Reply:
Does this still apply?